Transportation/BikeHire/doc/spec.md

```diff
@@ -0,0 +1,193 @@
+Bike Hire Docking Station
```

Comment:
are you sure  you are using markdown format as it is being used in the specification of other data models?